Header header

geometry_msgs/Point position
geometry_msgs/Quaternion orientation
geometry_msgs/Twist twist
geometry_msgs/Wrench wrench
geometry_msgs/Wrench external_wrench
float64 angle_self